contributors/design-proposals/resource-management/device-plugin.md

```diff
@@ -61,51 +61,72 @@ the following simple steps:
 
 # Use Cases
 
- * I want to use a particular device type (GPU, InfiniBand, FPGA, etc.)
-   in my pod.
- * I should be able to use that device without writing custom Kubernetes code.
+ * As a cluster admin I want my cluster to be aware of the devices on my nodes:
+    * My nodes should report what devices are present.
+    * My users should be able to request "device aware containers" (GPU,
+      InfiniBand, FPGA, etc...).
+    * My monitoring tool should be able to report device metrics.
+
+ * I should be able to enable my cluster to be aware of a device without
+   writing custom Kubernetes code.
```

Comment:
Another way to phrase this `I should be able to enable access to devices in kubernetes by extending it rather than forking it`